ndk-patches/linux-fcntl.h.patch

```diff
@@ -12,5 +11,5 @@ diff -uNr sysroot.orig/usr/include/linux/fcntl.h sysroot/usr/include/linux/fcntl
 +   https://android.googlesource.com/platform/bionic/+/3577825%5E!/      */
 +#define AT_EACCESS 0
  #define AT_SYMLINK_NOFOLLOW 0x100
+ #define AT_EACCESS 0x200
```

Comment:
AT_EACCESS is redefined here, meaning this patch does nothing

Reply:
Thanks for spotting that!